src/main/java/seedu/foodrem/logic/commands/generalcommands/HelpCommand.java

```diff
@@ -11,28 +12,21 @@
  * Format full help instructions for every command for display.
  */
 public class HelpCommand extends Command {
-
-    public static final String COMMAND_WORD = CommandWord.HELP_COMMAND.getValue();
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Displays help for FoodRem.\n"
-            + "Example: " + COMMAND_WORD;
-    public static final String SHOWING_HELP_MESSAGE = "Opened help window.";
     public static final String DEFAULT_HELP_MESSAGE = "Please refer to the user guide.";
 
     // To be displayed in a new help window
-    // TODO: Update this to match user guide
-    public static final String USER_GUIDE_URL = "https://se-education.org/addressbook-level3/UserGuide.html";
-    public static final String MORE_INFORMATION = "For more information please head to:\n" + USER_GUIDE_URL;
-
-    public static final String HELP_FOR_ALL_COMMANDS =
+    public static final String USER_GUIDE_URL = "https://ay2223s1-cs2103t-w16-2.github.io/tp/UserGuide";
+    private static final String MORE_INFORMATION = "For more information please head to:\n" + USER_GUIDE_URL;
+    private static final String COMMAND_WORD = HELP_COMMAND.getCommandWord();
+    private static final String HELP_FORMAT_GENERAL =
             "To receive help for a specific command, enter "
-                    + "\"" + HelpCommand.COMMAND_WORD + " COMMAND_WORD\" "
+                    + "\"" + COMMAND_WORD + " COMMAND_WORD\" "
                     + "in the command box, where COMMAND_WORD is any one of the following:\n"
-                    + CommandWord.listAllCommandWords() + "\n\n"
+                    + CommandType.listAllCommandWords() + "\n\n"
                     + MORE_INFORMATION;
-
-    public static final String NOT_A_COMMAND = "\"%s\" is not a valid command\n\n" + HELP_FOR_ALL_COMMANDS;
-
-    public static final String HELP_FOR_SPECIFIC_COMMAND = "%s\n\n" + HELP_FOR_ALL_COMMANDS;
+    public static final String NOT_A_COMMAND = "\"%s\" is not a valid command\n\n" + HELP_FORMAT_GENERAL;
+    private static final String SHOWING_HELP_MESSAGE = "Opened help window.";
```

Comment:
Small thing, but considering other commands has a clear success message defined, perhaps this can be renamed to SUCCESS_MESSAGE

Reply:
Agreed, this should be renamed. I suggest `FORMAT_COMMAND_HELP` since it's a format string, not an actual message, following similar conventions with other format strings defined elsewhere.